### PR TITLE
Fixing metadata

### PR DESCRIPTION
**Pull request by jbosse**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -44,21 +44,21 @@ jobs:
         id: npm-ci-test
         run: npm run ci-test
 
-  test-action:
-    name: GitHub Actions Test
-    runs-on: ubuntu-latest
+  # test-action:
```
Comment (jbosse) [inline .github/workflows/ci.yml:47]: I'm not sure how to integration test the action without real endpoints.

File: action.yml [hunk 2]
```diff
@@ -1,18 +1,48 @@
-name: 'The name of your action here'
-description: 'Provide a description here'
-author: 'Your name or organization here'
+name: 'dependabot-tickets'
```
Comment (jbosse) [inline action.yml:1]: I didn't even notice this file before.

File: src/planview-client.js [hunk 3]
```diff
@@ -29,15 +29,15 @@ class PlanviewClient {
   }
 
   #parseResponse(result) {
-    if (result.status === 200 && !!result.data.id) {
+    if (result.status === 201 && !!result.data.id) {
```
Comment (jbosse) [inline src/planview-client.js:32]: I think this was causing it to return the else condition to the Action runner so it was falling even though the ticket was created.

File: src/planview-client.js [hunk 4]
```diff
@@ -29,15 +29,15 @@ class PlanviewClient {
...
       return {
         success: true,
         data: result.data
       }
     } else {
       return {
         success: false,
-        result
+        data: result.data
```
Comment (jbosse) [inline src/planview-client.js:40]: I think result was causing an error because the Axios response is circular.

**Review by DBugger32 — APPROVED** [review]

**Pull request merged**